StorageTests/TestSQLiteHistory.swift

```diff
@@ -1292,7 +1342,8 @@ class TestSQLiteHistory: XCTestCase {
 
             return history.getTopSitesWithLimit(20) >>== { topSites in
                 XCTAssertEqual(topSites.count, 20)
-                XCTAssertEqual(topSites[0]!.guid, "abc\(0)def")
+                XCTAssertEqual(topSites[0]!.guid, "abc\(5)def")
+                XCTAssertEqual(topSites[1]!.guid, "abc\(0)def")
```

Comment:
nvm. There actually isn't a tie here. If you look at how we create fake history. https://github.com/mozilla-mobile/firefox-ios/blob/e83a50d86cfff646ba84155da629ba6d1907bb6e/StorageTests/TestSQLiteHistory.swift#L1461 We use the site number to advance the history timestamp for each site. So every site will have a unique timestamp.